### PR TITLE
Feature/header new

### PR DESCRIPTION
**Pull request by nvolodymyr**

### CODE REVIEW

File: src/app/component/core/header-new/header-new.component.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          <source media="(max-width: 576px)"
+                  srcset="assets/img/logo_S.png">
+          <img src="assets/img/logo_M.png" alt="logo">
+        </picture>
+        </a>
+      </div>
+<!--      <input class="menu-btn" type="checkbox" id="menu-btn" />-->
+<!--      <label class="menu-icon" for="menu-btn"-->
+<!--      ><span class="navicon"></span-->
+<!--      ></label>-->
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.html:16]: Please, remove commented code

File: src/app/component/core/header-new/header-new.component.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        </li>
+                      </ul>
+                    </nav>
+                  </div>
+        <div class="navigation-menu-right">
+              <ul>
+                          <li class="search"><a  href="#">    <img src="assets/img/icon/search.png" alt="search">  </a></li>
+              <li>    <div id="switcher-wrapper">
+                <img id="arrow" [src]="arrow" alt="v">
+                <select [(ngModel)]="language" (change)="changeCurrentLanguage()" id="language-switcher">
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.html:67]: Please, if you have more than 1 attribute - write them in separate rows

File: src/app/component/core/header-new/header-new.component.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                  <option class="language-option" [ngValue]="'uk'">Ua</option>
+                  <option class="language-option" [ngValue]="'ru'">Ru</option>
+                  <option class="language-option" [ngValue]="'en'">En</option>
+                </select>
+              </div>
+              </li>
+        <li class="sign-in-link"><a href="#">Sign in</a></li>
+    <li> <div id="create-button" class="secondary-global-button">
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.html:75]: Genereal: please, if you have more than 1 attribute - write them in separate rows

File: src/app/component/core/header-new/header-new.component.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import {UserSettingComponent} from '../../user/user-setting/user-setting.component';
+import {Router} from '@angular/router';
+import {LocalStorageService} from '../../../service/localstorage/local-storage.service';
+import {JwtService} from '../../../service/jwt/jwt.service';
+import {UserService} from 'src/app/service/user/user.service';
+import {AchievementService} from 'src/app/service/achievement/achievement.service';
+import {HabitStatisticService} from 'src/app/service/habit-statistic/habit-statistic.service';
+import {filter} from 'rxjs/operators';
+import {LanguageService} from '../../../i18n/language.service';
+import {Language} from '../../../i18n/Language';
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.ts:16]: General: write spaces between brackets and import name

File: src/app/component/core/header-new/header-new.component.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+export class HeaderNewComponent implements OnInit {
+  readonly notificationIcon = 'assets/img/notification-icon.png';
+  readonly userAvatar = 'assets/img/user-avatar.png';
+  readonly arrow = 'assets/img/arrow_grey.png';
+  dropdownVisible: boolean;
+  firstName: string;
+  userRole: string;
+  userId: number;
+  isLoggedIn: boolean;
+  language: string;
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.ts:33]: Genereal: please, write access modificators(private, public)

File: src/app/component/core/header-new/header-new.component.ts [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+              private localStorageService: LocalStorageService,
+              private jwtService: JwtService,
+              private router: Router,
+              private userService: UserService,
+              private achievementService: AchievementService,
+              private habitStatisticService: HabitStatisticService,
+              private languageService: LanguageService) {
+}
+
+  ngOnInit(): void {
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.ts:47]: Please, delete the type. It is not necessary in hooks

File: src/app/component/core/header-new/header-new.component.ts [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+        this.userId = userId;
+        this.isLoggedIn = true;
+      });
+    this.userRole = this.jwtService.getUserRole();
+    this.language = this.languageService.getCurrentLanguage();
+  }
+  public changeCurrentLanguage() {
+    this.languageService.changeCurrentLanguage(this.language as Language);
+  }
+  getUserId(): number | string {
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.ts:64]: Please, add access modificator

File: src/app/component/core/header-new/header-new.component.ts [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+        filter(userId => userId !== null && !isNaN(userId))
+      )
+      .subscribe(userId => {
+        this.userId = userId;
+        this.isLoggedIn = true;
+      });
+    this.userRole = this.jwtService.getUserRole();
+    this.language = this.languageService.getCurrentLanguage();
+  }
+  public changeCurrentLanguage() {
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.ts:61]: Please, add type of function

File: src/app/component/core/header-new/header-new.component.ts [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+  ngOnInit(): void {
+    this.dropdownVisible = false;
+    this.localStorageService.firstNameBehaviourSubject.subscribe(firstName => this.firstName = firstName);
+    this.localStorageService.userIdBehaviourSubject
+      .pipe(
+        filter(userId => userId !== null && !isNaN(userId))
+      )
+      .subscribe(userId => {
+        this.userId = userId;
+        this.isLoggedIn = true;
```
Comment (RomanDeveloperAcc) [inline src/app/component/core/header-new/header-new.component.ts:56]: Please, write it in a separate function

File: src/app/component/core/header-new/header-new.component.html (hunk 1 above)
Comment (nvolodymyr) [reply src/app/component/core/header-new/header-new.component.html:16]: +

File: src/app/component/core/header-new/header-new.component.ts (hunk 4 above)
Comment (nvolodymyr) [reply src/app/component/core/header-new/header-new.component.ts:16]: +

File: src/app/component/core/header-new/header-new.component.ts (hunk 5 above)
Comment (nvolodymyr) [reply src/app/component/core/header-new/header-new.component.ts:33]: +

File: src/app/component/core/header-new/header-new.component.ts (hunk 6 above)
Comment (nvolodymyr) [reply src/app/component/core/header-new/header-new.component.ts:47]: +

File: src/app/component/core/header-new/header-new.component.ts (hunk 7 above)
Comment (nvolodymyr) [reply src/app/component/core/header-new/header-new.component.ts:64]: +

File: src/app/component/core/header-new/header-new.component.ts (hunk 8 above)
Comment (nvolodymyr) [reply src/app/component/core/header-new/header-new.component.ts:61]: +

File: src/app/component/core/header-new/header-new.component.ts (hunk 9 above)
Comment (nvolodymyr) [reply src/app/component/core/header-new/header-new.component.ts]: +

**Pull request merged**